lookout/style/__init__.py

```diff
@@ -1,2 +1,3 @@
 """Supported machine learning based analyzers for lookout."""
+__version__ = 0, 0, 3
```

Comment:
Can you also please remove `__meta__` because damn it is only needed in the tests as far as i remember

Reply:
Shit we need to document that